### PR TITLE
V2 add global styles

### PR DESCRIPTION
**Pull request by anxolin**
Break down #1591 

This is the first of a series of PRs aiming to break down #1591 

The PR adds a Global Style (basically adding some defaults), adds some variables for the colors, and leaves some placeholders for future components that will come in different PRs.

### CODE REVIEW

File: src/components/layout/TradingLayout/variablesCss.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { css } from 'styled-components'
+
+const AllColors = `
```
Comment (anxolin) [inline src/components/layout/TradingLayout/variablesCss.ts:3]: This is a copy pasta from v1. @biocom do we need all this?
Maybe we can start smaller and add on need?

Also, I kind of like better the approach of uniswap on how they define the colors, and having 2 global styles, one for dark/light and one is just the defaults/css-reset

Comment (michelbio) [reply src/components/layout/TradingLayout/variablesCss.ts:3]: > do we need all this?
We might. I'd suggest to just get the file in place and then decide what we need/how we want to structure colors while working on the rest of the components.

**Review by michelbio — APPROVED** [review]

**Review by Velenir — APPROVED** [review]

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1596--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1596--dexreact.review.gnosisdev.com/storybook/)**: Component stories

**Review by biocom — APPROVED** [review]

File: src/components/layout/TradingLayout/variablesCss.ts (hunk 1 above)
Comment (biocom) [reply src/components/layout/TradingLayout/variablesCss.ts:3]: > do we need all this?

We might. I'd suggest to just get the file in place and then decide what we need/how we want to structure colors while working on the rest of the components.

**Review by W3stside — APPROVED** [review]

**Pull request merged**